ui/components/ui/box/box.d.ts

```diff
@@ -125,6 +130,7 @@ export interface BoxProps {
   style?: React.CSSProperties;
   as?: keyof HTMLElementTagNameMap;
   color?: TextColor | IconColor | string; // TODO: remove string when someone smarter figures out the issue with the color prop
+  ref?: RefObject<HTMLElement>;
 }
 
 declare const Box: React.FC<BoxProps>;
```

Comment:
Have another PR that will overwrite this one https://github.com/MetaMask/metamask-extension/pull/18160